src/sentry/tasks/post_process.py

```diff
@@ -512,19 +517,23 @@ def post_process_group(
         from sentry.models.project import Project
         from sentry.reprocessing2 import is_reprocessed_event
 
+        if eventstream_type == EventStreamEventType.Transaction.value:
+            processing_store = transaction_processing_store
+        else:
+            processing_store = event_processing_store
```

Comment:
Should this be behing a feature flag?

Reply:
that'll come after, for now the transaction_processing_store points to the event_processing_store. 
https://github.com/getsentry/getsentry/pull/15373